health_check/templates/health_check/index.html

```diff
@@ -1,32 +1,40 @@
 <html>
 <head>
-  <title>System status</title>
+  <title>
+    {% block title %}
+      System status
+    {% endblock title %}
+  </title>
```

Comment:
I think the title-tag innerHTML should not contain any newlines. Maybe something like this:
```suggestion
  <title>{% block title %}System status{% endblock title %}</title>
```